src/SonarQube/sonarlint-visualstudioCSharp.ruleset

```diff
@@ -1,7 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <RuleSet Name="Rules for SonarQube" Description="This rule set was automatically generated from SonarQube." ToolsVersion="14.0">
-  <Rules AnalyzerId="SonarLint.CSharp" RuleNamespace="SonarLint.CSharp">
+  <Rules AnalyzerId="SonarAnalyzer.CSharp" RuleNamespace="SonarAnalyzer.CSharp">
```

Comment:
Resynched the ruleset (I explicitly avoided updating the analyzers since I think there's a ticker or something special to do once the new analyzer is being used (?)).